api/edge_api/identities/models.py

```diff
@@ -183,6 +183,7 @@ def save(self, user: FFAdminUser = None, master_api_key: MasterAPIKey = None):
                     "environment_api_key": self.environment_api_key,
                     "changes": changes,
                     "identity_uuid": str(self.identity_uuid),
+                    "identifier": self.identifier,
```

Comment:
@khvn26, I had to add the identifier here as I need it in the response to the FE. I believe we'll also need it to power identity override in local evaluation too so I think it's data worth having.